src/lib_c/x86_64-windows-msvc/c/winbase.cr

```diff
@@ -90,4 +90,16 @@ lib LibC
   fun GetEnvironmentStringsW : LPWCH
   fun FreeEnvironmentStringsW(lpszEnvironmentBlock : LPWCH) : BOOL
   fun SetEnvironmentVariableW(lpName : LPWSTR, lpValue : LPWSTR) : BOOL
+
+  MEM_COMMIT     = 0x00001000
+  MEM_RESERVE    = 0x00002000
+  MEM_RESET      = 0x00080000
+  MEM_RESET_UNDO = 0x01000000
+
+  fun VirtualAlloc(lpAddress : Void*, dwSize : SizeT, flAllocationType : DWORD, flProtect : DWORD) : Void*
```

Comment:
These two functions are actually in `memoryapi.h`, according to MSDN, not winbase